gen3-integration-tests/utils/gen3_admin_tasks.py

```diff
@@ -169,21 +171,57 @@ def run_gen3_job(
             raise Exception("Build number not found")
     # Local Helm Deployments
     elif os.getenv("GEN3_INSTANCE_TYPE") == "HELM_LOCAL":
-        # job_pod = f"{job_name}-{uuid.uuid4()}"
-        cmd = ["kubectl", "delete", "job", job_name]
-        result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        if not result.returncode == 0:
-            logger.info(
-                f"Unable to delete {job_name} - {result.stderr.decode('utf-8')}"
+        if job_name == "fence-delete-expired-clients":
+            global fence_delete_clients_logs
+            # Get the pod name for fence app
+            cmd = ["kubectl", "get", "pods", "-l", "app=fence"]
+            result = subprocess.run(
+                cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True
             )
-        cmd = ["kubectl", "create", "job", f"--from=cronjob/{job_name}", job_name]
-        result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        if result.returncode == 0:
-            logger.info(f"{job_name} job triggered - {result.stdout.decode('utf-8')}")
-        else:
-            raise Exception(
-                f"{job_name} failed to start - {result.stderr.decode('utf-8')}"
+            if result.returncode == 0:
+                fence_pod_name = result.stdout.splitlines()[-1].split()[0]
```

Comment:
we can directly do
```
kubectl exec -it $(kubectl get pods -l app=fence -o jsonpath='{.items[0].metadata.name}') -- fence-create client-delete-expired
```
instead of capturing pod name separately, there is a risk of pod being deleted before the command is run with our dynamic environments